.sync/Files.yml

```diff
@@ -0,0 +1,232 @@
+# Specifies the files synced from Mu DevOps to other Project Mu repositories.
+#
+# This file is meant to ensure common configuration across Project Mu repos that is centralized in Mu DevOps.
+#
+# Files are contributed directly in Mu DevOps by the original author and then automatically synced after the PR
+# in Mu DevOps is merged by UEFI Bot to all of the repos specified in this file.
+#
+# To maintain consistency across Project Mu, always consider if a configuration/settings file is appropriate as
+# a common file in Mu DevOps or should be repo-specific.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# For more information, see:
+# https://github.com/BetaHuhn/repo-file-sync-action
+
+# Maintenance: Keep labels groups in ascending alphabetical order - easier to scan, identify duplicates, etc.
+
+# Azure Pipelines - Common Configs
+#
+# These files leverage a high degree of common build template reuse from Mu DevOps. It is preferred that repos
+# use the common templates for pipelines and adjust template parameter inputs as needed for a specific repo. However,
+# a repo can always be removed from this list and manage its pipeline YAML files completely independently if absolutely
+# necessary.
+#
+# Note: The fact that these files are copied to a repo does not mean that the repo has pipelines
+#       set up to use the YAML file. The file is simply available for pipelines to be connected
+#       if needed.
+group:

```

Comment:
Should we sync the license file?

Reply:
I thought about but backed off because mu_basecore has a TianoCore license file. Honestly, it's not a big deal though since we can skip that repo for the sync. I'll go ahead and add it.

Reply:
Added in PR commit https://github.com/microsoft/mu_devops/pull/32/commits/c0bd85a08ff79d5ea10835cfa471bbfdcce3d40d